transcom/mymove
# INT-B-20198

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A962842&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Updated the 858 generation to use emplid for coast guard

## How to test

1.  Create two users (one a coast guard and one non coast guard) and complete payment requests on both.
2. in terminal run this command for both:
`go run ./cmd/generate-payment-request-edi/ --payment-request-number {paymentRequestNumber}`
3. coast guard should have emplid in the 858 and non coast guard will have dodid
